NeoMathEngine/src/MathEngineDnnDropout.cpp

```diff
@@ -34,7 +34,21 @@ static inline int getMaskSize( float rate, bool isSpatial, bool isBatchwise, con
 	return batchWidth * objectSize;
 }
 
-CMathEngineDropoutDesc::CMathEngineDropoutDesc( IMathEngine& mathEngine, float rate, bool isSpatial, bool isBatchwise,
+static inline int getCacheSize(float rate, bool isSpatial, bool isBatchwise, const CBlobDesc& input)
+{
+	const int cacheSize = 64;
```

Comment:
Move `const int cacheSize = 64;` to `static constexpr int cacheSize = 64;` into the class CSeedDropoutDesc field, to inform about this parameter